tests/jquery.flot.touchNavigate.Test.js

```diff
@@ -671,7 +671,7 @@ describe("flot touch navigate plugin", function () {
               ]
           ], {
               xaxes: [{ autoScale: 'exact', plotPan: false }],
-              yaxes: [{ autoScale: 'exact' }],
+              yaxes: [{ autoScale: 'exact', plotPan: false }],
```

Comment:
I found the test tricky during developing. The y-axis is expected to be drag while its plotPan is default, but the test still passes. The root cause is the simulate touch move will also trigger 'doubletap', and 'doubletap' will recenter the axes.

After all, I update the test to, at least, make its settings and expectation match its description. However, this test will always pass even the plotPan setting are set to true. The other pan tests seem to have the same issue, they only expect some unchanged values but not the changed values. I guess the reason is that these tests are written with the 'recenter' problem.

BTW, we need a new API to disable the double-tap-recenter behavior. I think I can improve all these tests by disabling the 'recenter' behavior after that.

Reply:
Add Issue #280 for it.